src/App.tsx

```diff
@@ -1,5 +1,23 @@
+import { default as ColorData } from '../public/colors.json';
+import { ColorDisplay } from './ColorDisplay';
+import { type Color, filterColors } from './utils';
+
+const PAGE_MESSAGE = 'Color palette';
+
+// Sanitize data for duplicate values
+const COLOR_DATA = filterColors(ColorData);
+
 const App = () => {
-    return <h1>Hello World</h1>;
+    const colors = (COLOR_DATA as Color[]).map((color: Color) => {
```

Comment:
This ends up as a long long list in the UI - you could also display this as a grid or in columns, to make it a bit more user-friendly.